src/index.ts

```diff
@@ -9,6 +9,8 @@ export {
 	ButtonGroupOption,
 } from './components/ButtonGroup';
 export { Card, CardProps } from './components/Card';
+export { Changelog, ChangelogProps } from './components/Changelog';
+export { default as processChangelog } from './extra/Changelog/helpers.js';
```

Comment:
The point of having it in the `extra`s is to not export it via the `index.ts`.
Let's either remove it from here, or move it outside the `extra`s.